programs/python-tensorflow.json

```
@@ -0,0 +1,10 @@
+{
+    "files": [
+        {
+            "help": "Export the following environment variables:\n\n```bash\nexport KERAS_HOME=\"${XDG_STATE_HOME}/keras\"\n```\n\nThis looks untested however.\n_Relevant issue_: https://wiki.archlinux.org/title/XDG_Base_Directory\n",
```

```suggestion
            "help": "Export the following environment variables:\n\n```bash\nexport KERAS_HOME=\"${XDG_STATE_HOME}/keras\"\n```\n\nThis looks untested however.\n_Relevant issue_: https://github.com/tensorflow/tensorflow/issues/38831\n",
```